openedx/core/djangoapps/schedules/tests/test_resolvers.py

```diff
@@ -274,7 +274,7 @@ def create_resolver(self, user_start_date_offset=8):
     def test_schedule_context(self):
         resolver = self.create_resolver()
         # using this to make sure the select_related stays intact
-        with self.assertNumQueries(30):
+        with self.assertNumQueries(33):
```

Comment:
Is this related?

Reply:
The waffle flag introduces extra queries. Not really sure how/why the addition in the unit render pipeline shows up in this way, but that is why the increase.